integrations/facebook-pixel/lib/index.js

```diff
@@ -52,6 +54,9 @@ FacebookPixel.prototype.initialize = function() {
   window.fbq.version = '2.0';
   window.fbq.queue = [];
   this.load(this.ready);
+  if (!this.options.automaticConfiguration) {
```

Comment:
This conditional is relevant to [PLATFORM-2310](https://segment.atlassian.net/browse/PLATFORM-2310).

All changes below this belong to [PLATFORM-2538](https://segment.atlassian.net/browse/PLATFORM-2538)